src/modules/Storage.rb

```diff
@@ -5021,6 +5021,7 @@ def AddSubvolRoot(part)
         "var/lib/libvirt/images",
         "var/lib/mailman",
         "var/lib/mariadb",
+        "var/lib/mysql",
```

Comment:
/var/lib/mariadb was explicitly demanded in that underlying feature request, so who are we to simply remove it... ;-)